lib/elixir/src/elixir_tokenizer.erl

```diff
@@ -132,6 +132,12 @@ tokenize([], EndLine, _Column, #elixir_tokenizer{terminators=[{Start, {StartLine
   Message = io_lib:format("missing terminator: ~ts (for \"~ts\" starting at line ~B)", [End, Start, StartLine]),
   {error, {EndLine, Message, []}, [], Tokens};
 
+% ## VC merge conflict
```

Comment:
I was just following the style in the rest of the file, I can change all occurences